sdv/multi_table/hma.py

```diff
@@ -144,6 +146,37 @@ def _estimate_columns_traversal(self, table_name, columns_per_table, visited):
 
         visited.add(table_name)
 
+    def _print_estimate_warning(self):
+        total_est_cols = 0
+        metadata_columns = self._get_num_data_columns()
+        print_table = []
+        for table, est_cols in self._estimate_num_columns().items():
+            entry = []
+            entry.append(table)
+            entry.append(metadata_columns[table])
+            total_est_cols += est_cols
+            entry.append(est_cols)
+            print_table.append(entry)
+        if total_est_cols > 1000:
+            self._print(
+                'PerformanceAlert: Using the HMASynthesizer on this metadata '
+                'schema is not recommended. To model this data, HMA will '
+                f'generate a large number of columns. ({total_est_cols} columns)\n\n')
+            self._print(
+                tabulate(
+                    print_table,
```

Comment:
I think if you convert the `print_table` to a dataframe it should print nicely/ there should be a way to print it. That way we don't have to add the tabulate dependency